.deployment/roles/opencast/templates/bin/setenv

```diff
@@ -34,7 +34,7 @@
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
 #
 # Debug options
-# export KARAF_DEBUG       # Enable debug mode
+export KARAF_DEBUG=true       # Enable debug mode
```

Comment:
Is this intentional? :thinking:

Reply:
Yes. It's in a separate commit. Or what do you mean?

Reply:
Yeah no makes sense. Somehow I only registered one commit. :eyes: